backend/server.js

```diff
@@ -4,6 +4,7 @@ import config from 'config'
 
 const app = Fastify({
   logger: true,
+    bodyLimit: 10048576
```

Comment:
@AntonW I would prefer not to have that hard-coded here. Especially, not the "random" value of 10 MiB. The standard [suggests to use 15 MiB](https://docs.oasis-open.org/csaf/csaf/v2.0/os/csaf-v2.0-os.html#c1-file-size).

Reply:
@AntonW Thanks for changing it.